app/views/govuk_publishing_components/components/docs/intervention.yml

```diff
@@ -33,9 +36,11 @@ examples:
     data:
       suggestion_text: You should renew your permit every 6 months.
       dismiss_text: Hide this suggestion
+      name: campaign-name
 
   with_suggestion_link_only:
     data:
+      suggestion_text: You should renew your permit every 6 months.
```

Comment:
I'm confused why `suggestion_text` has been added to all of these examples, can you explain why it wasn't here before? I think for this specific example it's been added in error - the example name suggests that this is to show the component without suggestion text.

Reply:
It was missing previously as the examples only showed banners with a dismiss link with no text... which was useless.

Reply:
If that's the case this example should be removed entirely. Also should `suggestion_text` be a required option for the component to render?

Reply:
Addressed in https://github.com/alphagov/govuk_publishing_components/pull/3101/commits/5bdf8c768d8b7b7ad5a39b45f38dd80f11561483